src/libs/actions/OnyxUpdates.ts

```diff
@@ -119,6 +119,9 @@ function apply({lastUpdateID, type, request, response, updates}: OnyxUpdatesFrom
  * @param [updateParams.updates] Exists if updateParams.type === 'pusher'
  */
 function saveUpdateInformation(updateParams: OnyxUpdatesFromServer) {
+    // Pause the Sequential queue so we don't do any other requests while we're fetching missing onyxUpdates
+    SequentialQueue.unpause();
```

Comment:
The comment says to pause the queue, but this is unpausing the queue 😓

Reply:
Wrong comment, should be written in pause.